config/routes.rb

```diff
@@ -6,6 +6,15 @@
 
   root "users#index" # Defines the root path route ("/")
 
+  # The following routes are for the API.
+  namespace :api do
+    namespace :v1 do
+      resources :posts, only: [:index]
+      resources :comments, only: [:index, :create]
+    end
+  end
```

Comment:
[REQUIRED] I noticed that you didn't make the changes suggested by the previous reviewer probably because you didn't understand his suggestion so I will try to explain it more clearly. In the image below you will notice a list of requirements you are meant to fulfill for the API.
![image](https://github.com/lily4178993/nell-blog/assets/62938735/03b8e224-ecf2-49ff-8a92-a3c5961c30f2)
 This includes:
1. An endpoint that lists all posts for a user. Your endpoints list all posts that exist instead of those belonging to a user. To achieve this you will need to nest your resources. For example:
```
resources :users do
    resources :posts, only: [:index]
 end
```
This creates your routes like `users/:user_id/posts` instead of `posts`.

2. An API endpoint that lists all comments for a user's posts and another that creates comments. Just like the previous requirement, you will need to nest the routes. So you make sure that posts resources are also a block just like users in the example above and you add comments to it. So user wraps posts and posts wrap comments.
You can checkout this [link](https://www.digitalocean.com/community/tutorials/how-to-create-nested-resources-for-a-ruby-on-rails-application) for more on nested routes

Reply:
Thanks for your feedbacks. I've corrected that